scripts/convert_hf_checkpoint.py

```diff
@@ -0,0 +1,95 @@
+from pathlib import Path
+
+from transformers import LlamaForCausalLM
+import torch
+
+from lit_llama.model import LLaMA, LLaMAConfig
+
+
+# Perform the reverse operation of: https://github.com/huggingface/transformers/blob/main/src/transformers/models/llama/convert_llama_weights_to_hf.py
```

Comment:
Should this comment be attached to the function?

Reply:
Done